documentation-site/static/examples/select/open.js

```diff
@@ -0,0 +1,24 @@
+import * as React from 'react';
+import {StatefulSelect} from 'baseui/select';
+
+export default () => (
+  <StatefulSelect
+    options={[
+      {id: 'AliceBlue', color: '#F0F8FF'},
+      {id: 'AntiqueWhite', color: '#FAEBD7'},
+      {id: 'Aqua', color: '#00FFFF'},
+      {id: 'Aquamarine', color: '#7FFFD4'},
+      {id: 'Azure', color: '#F0FFFF'},
+      {id: 'Beige', color: '#F5F5DC'},
+    ]}
+    labelKey="id"
+    valueKey="color"
+    overrides={{
+      Root: {
+        // Push the next example down so the dropdown does not block it.
```

Comment:
let's remove this from the example, and add it to the markdown file. i think you can add an arbitrary div with a style prop